config.js

```diff
@@ -70,5 +70,6 @@ module.exports = {
   PRETTY_DATE_FORMAT: 'Do MMMM YYYY',
   dateTimeFormat: 'DD MMM YYYY HH:mm:ss',
   dobFormat: 'YYYY-MM-DD',
-  dobCutoff: '2003-08-23'
+  // dbCutoff is the earliest date of birth that qualifies as under 18, therefore 27 August 2003 is considered over 18
```

Comment:
not hugely important but typo: `dbCutoff` -> `dobCutoff`

Reply:
EagleEye +1